src/views/Login/components/OnboardingStepper/components/2.BackupSeedStep/BackupSeedStep.test.tsx

```diff
@@ -34,11 +32,29 @@ it("should have appropriate checkbox behavior", () => {
   expect(checkboxClickSpy).toHaveBeenCalledTimes(1);
 });
 
+it("should regenerate the accountSeed when the refresh button is clicked", () => {
```

Comment:
is this test now mostly obsolete due to cypress?

Reply:
probably, I put a broad TODO in to review test applicability. for now these tests still pass so there shouldn't be any harm in retaining them